### ISSUE TITLE
[OJS] current release contains potentially dangerous old version of PHPMailer

### ISSUE THREAD

**Issue by woodfighter** [pkp/pkp-lib#3392]
The version of PHPMailer packaged in the Current Production Release <http://pkp.sfu.ca/ojs/download/ojs-3.1.0-1.tar.gz> is 5.2.8 and nearly 4 years old. That version is vulnerable to [several security issues](https://github.com/PHPMailer/PHPMailer/blob/master/SECURITY.md), I haven't tried to exploit them via OJS but it's at least a possibility.

**Comment by asmecher** [comment]
Thanks for the heads-up, @woodfighter. At a quick glance, I *believe* we're OK, because of regexps we run over incoming emails. But the best solution will definitely be to update this library (and other Composer dependencies that have been kept back). I'll look into it ASAP.

**Comment by asmecher** [comment]
I've updated the PHPMailer version to `^5.2` (=currently 5.2.26) in `master` and `ojs-stable-3_1_0`. I've also filed https://github.com/pkp/pkp-lib/issues/3404 for a more general review to make sure we're not at risk of this sort of problem in the future. Thanks for reporting!

If you want to update your existing install to use a newer (and still compatible) PHPMailer, apply the change to `lib/pkp/composer.json` described in https://github.com/pkp/pkp-lib/commit/867f29265b8450b29d678df2540d1ede5a10e419, and then run `composer update`.
